embedchain/embedchain.py

```diff
@@ -188,11 +190,19 @@ def generate_prompt(self, input_query, context):
         :param context: Similar documents to the query used as context.
         :return: The prompt
         """
-        prompt = f"""Use the following pieces of context to answer the query at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer.
-        {context}
-        Query: {input_query}
-        Helpful Answer:
-        """
+        if type(context) is list and len(context) > 0:
```

Comment:
```suggestion
        if type(contexts) is list and len(contexts) > 0:
```

Reply:
Since I accepted the other change, renaming to contexts has to be done, and is logical too. Accepted.